src/app/loadout/plug-drawer/PlugSection.tsx

```diff
@@ -9,40 +9,52 @@ import SelectablePlug from './SelectablePlug';
  * a list of plugs, plus some metadata about:
  * - the maximum we should let the user choose at once
  * - the plugset whence these plugs originate
+ * - the behavior we use for selecting plugs
  */
-export interface PlugsWithMaxSelectable {
+export interface PlugSet {
   /** The hash that links to the PlugSet definition. */
   plugSetHash: number;
   /** A list of plugs from this plugset. */
   plugs: PluggableInventoryItemDefinition[];
   /** The maximum number of plugs a user can select from this plug set. */
   maxSelectable: number;
+  /**
+   * The select behavior of the plug set.
+   * multi: how armour mods are selected in game, you need to manually remove ones that have been added.
+   * single: how abilities in subclasses are selected, selecting an option replaces the current one.
+   */
+  selectionType: 'multi' | 'single';
```

Comment:
Just to call it out, the main change was the addition of this thing. It allows us to determine the behaviour or the items when you select a new one. I.e. whether it adds a new one (multi) or it replaces the current one (single).